BBBBroadcast/Services/WebRTCClient.swift

```diff
@@ -160,33 +125,13 @@ final class WebRTCClient: NSObject {
     }
     
     private func createVideoTrack() -> RTCVideoTrack {
-        let videoSource = WebRTCClient.factory.videoSource()
-        
-        #if targetEnvironment(simulator)
-        self.videoCapturer = RTCFileVideoCapturer(delegate: videoSource)
-        #else
-        self.videoCapturer = RTCCameraVideoCapturer(delegate: videoSource)
-        #endif
-        
-        let videoTrack = WebRTCClient.factory.videoTrack(with: videoSource, trackId: "video0")
+        videoSource = WebRTCClient.factory.videoSource()
+        videoCapturer = RTCVideoCapturer(delegate: videoSource!)
+        videoSource!.adaptOutputFormat(toWidth: 441, height: 736, fps: 15)
```

Comment:
What are these magic numbers?

Reply:
These are hardcoded width and height for video output format. I added them because BroadcastExtension was crashing with default values which made output format too big and memory footprint went higher than allowed for BroadcastExtension, causing crash. So I experimented with format values and found these are optimal. This need to be adjusted later in accordance with BBB rules. So let it stay for now, it should be changed later.